packages/frontend/component/src/components/theme-provider/index.tsx

```diff
@@ -1,12 +1,27 @@
-import { ThemeProvider as NextThemeProvider } from 'next-themes';
+import { LiveData } from '@toeverything/infra';
+import { ThemeProvider as NextThemeProvider, useTheme } from 'next-themes';
 import type { PropsWithChildren } from 'react';
+import { useEffect } from 'react';
 
 const themes = ['dark', 'light'];
 
+export const appTheme$ = new LiveData<string | undefined>(undefined);
```

Comment:
Put this value into framework

Reply:
Thanks for your advice! I added a `AppThemeService` and regist it on the framework.